bin/install-if-deps-outdated.js

```diff
@@ -51,15 +51,16 @@ function install() {
 		console.error( 'failed to clean: %o', cleanResult );
 		process.exit( cleanResult.status );
 	}
+
 	const installResult = spawnSync( 'npm', [ 'ci' ], {
 		shell: true,
 		stdio: 'inherit',
-	} ).status;
+	} );
 	if ( installResult.status ) {
 		console.error( 'failed to install: %o', installResult );
```

Comment:
Printing `installResult` here isn't very informative:

```
failed to install: {
  status: 1,
  signal: null,
  output: [ null, null, null, [length]: 3 ],
  pid: 20586,
  stdout: null,
  stderr: null
}
```

We already get a message, that may be enough:

```
npm ERR! cipm can only install packages with an existing package-lock.json or npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or later to generate it, then try again.
```

Reply:
That's true. I'd expect the error output to be in `result.stderr` instead of everything being `null` -- but that's probably a result of `stdio: 'inherit'` where all output is handled by the parent process' file descriptor.

Pushed a fix to log only the exit code.